include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -554,6 +572,25 @@ class argument_parser
     //!\brief Stores the sub-parser in case \link subcommand_arg_parse subcommand parsing \endlink is enabled.
     std::unique_ptr<argument_parser> sub_parser{nullptr};
 
+    /*!\brief The format of the argument parser that decides the behavior when
+     * calling the seqan3::argument_parser::parse function.
+     * \details The format is set in the function argument_parser::init.
+     */
+    std::variant<detail::format_parse,
```

Comment:
Is this the only place where the variant type is defined?

Reply:
Yes